src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
+    public static final String SUPPLY_OPERATION = "supply";
+    public static final String BUY_OPERATION = "buy";
+    public static final String RESULT = "result";
+    public static final String DATA_SEPARATOR = " ";
+    public static final String CSV_SEPARATOR = ",";
+    public static final int OPERATION_INDEX_0 = 0;
+    public static final int AMOUNT_INDEX_1 = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(createReport(readFile(fromFileName)), toFileName);
+    }
+
+    private String readFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
```

Comment:
For the future, try to avoid such naming(stringBuilder, var, values), how to guess what it stores